src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -2234,55 +2237,7 @@ .maxstack  2
 ", sequencePoints: "Program+<Test>d__0.MoveNext", source: source);
         }
 
-        [Fact]
-        [WorkItem(12564, "https://github.com/dotnet/roslyn/issues/12564")]
-        public void ConditionalInSimpleMethod()
```

Comment:
Why delete this test?

Reply:
That's a mistake. Thank you for catching.

---
In reply to: [211443158](https://github.com/dotnet/roslyn/pull/29381#discussion_r211443158) [](ancestors = 211443158)